shared/my-sites/themes/jetpack-manage-disabled-message.jsx

```diff
@@ -21,10 +23,37 @@ export default React.createClass( {
 		} ).isRequired
 	},
 
-	clickOnActivate: function() {
+	clickOnActivate() {
 		analytics.ga.recordEvent( 'Jetpack', 'Activate manage', 'Site', this.props.site ? this.props.site.ID : null );
 	},
 
+	getMockThemes() {
```

Comment:
Since this method actually returns a component tree, rather than theme objects, should it be named `renderMockThemesList`?